packages/notifier-seeder/test/notificationMessage.integration.test.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable functional/immutable-data */
+/* eslint-disable functional/no-let */
+import { StartedTestContainer, GenericContainer } from "testcontainers";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+
+import {
+  EServiceDescriptorSuspendedV2,
+  EServiceDescriptorV2,
+  EServiceEventEnvelopeV2,
+  EServiceV2,
+  descriptorState,
+  eserviceMode,
+  generateId,
+  technology,
+  toDescriptorV2,
+  toEServiceV2,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { v4 } from "uuid";
+import {
+  QueueManager,
+  initQueueManager,
+} from "../src/queue-manager/queueManager.js";
+import { toCatalogItemEventNotification } from "../src/models/catalogItemEventNotificationConverter.js";
+import { buildCatalogMessage } from "../src/models/catalogItemEventNotificationMessage.js";
+
+import catalog_item_descriptor_updated from "./resources/catalogItemDescriptorUpdate.json" assert { type: "json" };
+
+export const TEST_ELASTIC_MQ_IMAGE = "softwaremill/elasticmq-native:latest";
+export const TEST_ELASTIC_MQ_PORT = 9324;
+
+export const elasticMQContainer = (): GenericContainer =>
+  new GenericContainer(TEST_ELASTIC_MQ_IMAGE)
+    .withCopyFilesToContainer([
+      {
+        source: "elasticmq.local.conf",
+        target: "/opt/elasticmq.conf",
+      },
+    ])
+    .withExposedPorts(TEST_ELASTIC_MQ_PORT);
```

Comment:
This is the same used in `queueManager` tests, it could be moved to a utils file.
Also, I just noticed that the container tag was not pinned to a specific version like the other ones, but it's using `latest` (my bad, since I set this up :)) - in this other PR I'm pinning it to version `1.5.7` in the docker-compose, you could do the same here https://github.com/pagopa/interop-be-monorepo/pull/340/files#diff-423deb13b7c401b1a7f41ee91c77f722e11d2f317d6a66b546524e8a04cc8b03R124